src/common/hooks/auth/use-auth-request.ts

```diff
@@ -0,0 +1,6 @@
+import { useRecoilValue } from 'recoil';
+import { authRequestState } from '@store/onboarding';
+
+export function useAuthRequest() {
```

Comment:
Why do this if it is only used in one place?

Reply:
It's used in more than one place